### PR TITLE
Update `Snap geometry edits` readme to include default haptics information

### PR DESCRIPTION
**Pull request by duffh**
## Description

PR to update the Kotlin sample _"Snap geometry edits"_ in `Edit and Manage Data` category.

Included information on the default haptics experience, how to disable haptics and implement custom haptics based on `SnapState`.

### CODE REVIEW

File: snap-geometry-edits/README.md [hunk 1]
```diff
@@ -65,6 +65,8 @@ Snapping can be used during interactive edits that move existing vertices using
 
 Geometry guides are enabled by default when snapping is enabled. These allow for snapping to a point coinciding with, parallel to, perpendicular to, or extending an existing geometry.
 
+On supported platforms haptic feedback on `SnapState.snappedToFeature` and `SnapState.snappedToGeometryGuide` is enabled by default when snapping is enabled. Custom haptic feedback can be configured by setting `SnapSettings.isHapticFeedbackEnabled` to false and listening to `GeometryEditor.snapChanged` events to provide specific feedback depending on the `SnapState`.
```
Comment (TADraeseke) [inline snap-geometry-edits/README.md:68]:
```suggestion
On supported platforms, haptic feedback on `SnapState.snappedToFeature` and `SnapState.snappedToGeometryGuide` is enabled by default, when snapping is enabled. Custom haptic feedback can be configured by setting `SnapSettings.isHapticFeedbackEnabled` to false and listening to `GeometryEditor.snapChanged` events to provide specific feedback, depending on the `SnapState`.
```

Comment (TADraeseke) [reply snap-geometry-edits/README.md:68]: I wonder if we should use the term SharedFlow or Flow at any point here to make it more kotlin specific

**Review by TADraeseke — COMMENTED** [review]
Just a couple small bits!

**Comment by shubham7109** [comment]
@duffh  ⚠️Do not merge this PR `v.next` ⚠️ . We are in the process of replacing the project on `v.next` with the new sample viewer design. I have carried over these changes to https://github.com/Esri/arcgis-maps-sdk-kotlin-samples/pull/249, and I would recommend pointing to the `design/compose-sample-viewer` for future PRs which will eventually replace `v.next`.

**Review by shubham7109 — APPROVED** [review]
LGTM 👍

File: snap-geometry-edits/README.md (hunk 1 above)
Comment (TADraeseke) [reply snap-geometry-edits/README.md:68]: Here's an example of SharedFlow in doc: https://developers.arcgis.com/kotlin/api-reference/arcgis-maps-kotlin/com.arcgismaps.realtime/-custom-dynamic-entity-data-source/-entity-feed-provider/feed.html

Happy to skip commas, just a suggestion :D

**Pull request closed without merging**